application/provider.go

```diff
@@ -105,6 +106,12 @@ func NewProvider(prefix string, scopes ...string) (*ProviderImpl, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to create logger")
 	}
+	if os.Getenv("TIDEPOOL_LOGGER_PACKAGE") == "devlog" {
```

Comment:
Please simplify to something like:
```
    var logger log.Logger
    if os.Getenv("TIDEPOOL_LOGGER_PACKAGE") == "devlog" {
        logger, err = devlog.New(writer, log.DefaultLevelRanks(), log.Level(level))
    } else {
        logger, err = logJson.NewLogger(writer, log.DefaultLevelRanks(), log.Level(level))
    }
    if err != nil {
	    return nil, errors.Wrap(err, "unable to create logger")
    }
```
so it only creates one logger.

Reply:
I've switched to a switch (alliteration!) I wrote it the way I originally did to try to convey that the jsonLog was the default, and devlog was an auxiliary choice. Your suggestion (a very natural way to write the code) didn't capture that intention well. Which is why I didn't just do that in the first place.

With a switch it's clear that the default is json, so I'm happy—hopefully it's acceptable to you as well.